examples/main.go

```diff
@@ -23,15 +23,269 @@ func main() {
 		log.Fatalf("did not connect: %v", err)
 	}
 	defer conn.Close()
-	c := pb.NewQdrantClient(conn)
+
+	// create grpc collection client
+	collections_client := pb.NewCollectionsClient(conn)
 
 	// Contact the server and print out its response.
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
-	r, err := c.HealthCheck(ctx, &pb.HealthCheckRequest{})
+
+	// Check Qdrant version
+	qdrantClient := pb.NewQdrantClient(conn)
+	healthCheckResult, err := qdrantClient.HealthCheck(ctx, &pb.HealthCheckRequest{})
+	if err != nil {
+		log.Fatalf("Could not get health: %v", err)
+	} else {
+		log.Printf("Qdrant version: %s", healthCheckResult.GetVersion())
+	}
+
+	collectionName := "test_collection"
+
+	// Delete collection
+	_, err = collections_client.Delete(ctx, &pb.DeleteCollection{
+		CollectionName: collectionName,
+	})
+	if err != nil {
+		log.Fatalf("Could not delete collection: %v", err)
+	} else {
+		log.Println("Collection", collectionName, "deleted")
+	}
+
+	// Create new collection
+	var defaultSegmentNumber uint64 = 2
```

Comment:
same as above.

Reply:
I introduced this variable just to take pointer (proto wants pointer instead of value). So I want to keep here as is